site/gatsby-site/src/components/checklists/ChecklistsIndex.js

```diff
@@ -47,43 +122,56 @@ const ChecklistsIndex = ({ users }) => {
           <h1 className="mr-auto">
             <Trans>Risk Checklists</Trans>
           </h1>
-          {loggedIn && (
-            <Button
-              id="new-checklist-button"
-              onClick={async () => {
-                const newChecklist = {
-                  id: generateId(),
-                  owner_id: user.id,
-                  name: 'Unspecified System',
-                  about: '',
-                  tags_goals: [],
-                  tags_methods: [],
-                  tags_other: [],
-                  risks: [],
-                };
-
-                try {
-                  await insertChecklist({
-                    variables: { checklist: newChecklist },
-                  });
-                } catch (error) {
-                  addToast({
-                    message: t('Could not create checklist.'),
-                    severity: SEVERITY.danger,
-                    error,
-                  });
-                  return;
-                }
-                window.location = '/apps/checklists?id=' + newChecklist.id;
-              }}
-            >
-              <Trans>New</Trans>
-            </Button>
-          )}
+          <div className="flex gap-2 items-center">
+            <label htmlFor="sort-by">Sort</label>
+            <Select id="sort-by" onChange={(evt) => setSortBy(evt.target.value)}>
+              <option value="alphabetical">Alphabetical</option>
+              <option value="oldest-first">Oldest First</option>
+              <option value="newest-first">Newest First</option>
+              <option value="last-updated">Last Updated</option>
+            </Select>
```

Comment:
The text of this Select component overlaps the selector arrow
<img width="124" alt="image" src="https://github.com/responsible-ai-collaborative/aiid/assets/6564809/b08e951d-9651-4a12-873b-a3448d8860da">

Reply:
Fixed.

![image](https://github.com/responsible-ai-collaborative/aiid/assets/25443411/b739aaf5-ecb1-4f59-9b23-cd907e9e48f5)